CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [5.0.0](https://github.com/awslabs/fhir-works-on-aws-deployment/compare/v4.3.1...v5.0.0) (2022-07-22)
+
+
+### ⚠ BREAKING CHANGES
+
+* Move deployment pipeline to CDK (#654). Serverless deployments are still supported, but have been marked as legacy moving forwards.
```

Comment:
Can we add some more information here like for existing customer, if they pull down this release and deploy using serverless, they'll still get all the bugfixes included in this release?